includes/widgets/common.php

```diff
@@ -724,14 +724,16 @@ private function register_background_section() {
 			[
 				'label' => esc_html__( 'Transition Duration', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
-				'size_units' => [ 's', 'ms' ],
-				'default' => [
-					'unit' => 's',
+				'range' => [
+					'px' => [
+						'max' => 3,
+						'step' => 0.1,
+					],
 				],
 				'render_type' => 'ui',
 				'separator' => 'before',
 				'selectors' => [
-					'{{WRAPPER}} > .elementor-widget-container' => 'transition: background {{SIZE}}{{UNIT}}',
+					'{{WRAPPER}} > .elementor-widget-container' => 'transition: background {{SIZE}}ms',
```

Comment:
Shouldn't the label have "ms"?

Reply:
Done

Reply:
It was a question, IDK what was before that, just wanted to make sure. Did you check that?